settings/base.py

```diff
@@ -0,0 +1,160 @@
+"""
+Base settings for ORA2.
```

Comment:
Also this is looking a _lot_ like the ORA2 block, which is actually installed explicitly as a Django app. Do you intend for this XBlock to be installed as an XBlock, or as a Django app?